src/shims/windows/handle.rs

```diff
@@ -8,6 +9,10 @@ use crate::*;
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
 pub enum PseudoHandle {
     CurrentThread,
+    CurrentProcess,
+    Stdin,
+    Stdout,
+    Stderr,
```

Comment:
Note that the handles returned by `GetStdHandle` are proper file handles and not pseudo-handles. This differs from posix where the fds have fixed values. I realise that miri does want to track the difference but I worry about confusing the terminology.

Reply:
Hm, some ideas on paths forwards:
- Leave it as-is, but add documentation about the terminology here
- Try to merge stdin/out into `Handle::File` as normal `FileDescription` (basically what unix does)
  - This will require some more changes to the `FileDescription` trait. Its `read` and `write` methods are a bit tailored towards unix flavor currently, you may notice I have some hacky code in here to work around that. I'd like to fix it eventually anyways, but it adds more friction to MVP
- Make `Handle::File` hold an enum like `enum FileHandle { Stdin, Stdout, File(i32) }`

Reply:
I don't think miri actually needs to track the difference. It should be fine to use FileDescriptions for them like it does on Unix

Reply:
> Its read and write methods are a bit tailored towards unix flavor currently, you may notice I have some hacky code in here to work around that. I'd like to fix it eventually anyways, but it adds more friction to MVP

Is it even worth using the same trait for Unix and Windows? The trait is indeed very tailored towards Unix, deliberately so. Knowing nothing about the Windows APIs, I expected they would need their own trait anyway.

I haven't looked yet at those hacks. Can you write a high-level summary of the pros and cons of using the same vs a different trait?